### DISCUSSION TITLE
question about the biases estimation

### DISCUSSION THREAD

**Discussion by lmy91** [rodralez/NaveGo#147] [Q&A] [answered]
Dear team of Navego：
Recently, I was reading the Navego program and I have a question about sensor bias compensation. In the program the sensor biases are modeled as GM. And after reading your paper, I know there are the error of the sensor biases. However, I don't understand why the biases estimation is like this(see the attached picture). 
![BIAS ESTIMATION](https://user-images.githubusercontent.com/83980581/186406352-7ccac7e4-4cdc-4e46-a3c6-c49c845f02a7.png)
Is that means the dynamic biases are reset to zero when the next filtering starts?
Sorry to bother you and I'm looking forward to your reply.
Kind regards.

**Comment by rodralez** [comment, marked as answer]
Hello, 

Sorry for the late reply!

> Is that means the dynamic biases are reset to zero when the next filtering starts? 

No, dynamic bias states are not error Kalman states but direct Kalman states. Thus, the values of the dynamic bias states are estimated, not the errors between the truth values and the estimated values.
